src/page-multi-facility/CreateBaselineScenarioPage.tsx

```diff
@@ -47,11 +47,29 @@ const CreateBaselineScenarioPage: React.FC = () => {
       <ModalDialog open title="Welcome">
         <ModalContent>
           <WelcomeText>
-            Welcome to your new scenario. To get started, add in facility data
-            on the right-hand side of the page. Your initial scenario is also
-            your 'Baseline' - meaning this is where you should keep real-world
-            numbers about the current state of your facilities, their cases, and
-            mitigation steps.
+            This tool will help you model Covid-19 cases in a set of criminal
```

Comment:
In most other places I've seen on Recidiviz's web site, Covid-19 is referred to in all caps like this: COVID-19.  My spellchecker even marks the lower case version as incorrect

Reply:
Good catch!